draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5427,6 +5434,18 @@ Content-Range: exampleunit 11.2-14.3/25
    A cache that receives a payload in a GET request is likely to ignore that
    payload and cache regardless of the payload contents.
 </t>
+<t>
+   When information retrieval is performed with a mechanism that constructs a
+   target URI from user-provided information, such as the query fields of a
+   form using GET, potentially sensitive data might be provided that would not
+   be appropriate for disclosure within a URI
+   (see <xref target="sensitive.information.in.uris"/>). In some cases, the
+   data can be filtered or transformed such that it would not reveal such
+   information. In others, particularly when there is no benefit from caching
+   a response, using the POST method (<xref target="POST"/>) instead of GET
+   will transmit such information in the request body rather than construct a
+   new URI.
```

Comment:
```suggestion
   a response, using the POST method (<xref target="POST"/>) instead of GET
   will usually transmit such information in the request body rather than construct a
   new URI.
```